package.json

```
@@ -19,6 +19,15 @@
     "url": "https://github.com/chromaui/chromatic-cli.git"
   },
   "license": "MIT",
+  "exports": {
+    ".": {
+      "require": "./isChromatic.js",
+      "import": "./isChromatic.mjs"
+    },
+    "./api": {
+      "import": "./node/main.cjs"
```

while the docs on this topic has always been incredibly elusive, I think `import` is for ESM and `require is for CJS, so this looks strange to me. Although if you say it works. 🤷
```suggestion
      "require": "./node/main.cjs"
```

---

Oh lol, no I probably had it backwards. Node doesn't use these entry points so I didn't really test them yet :)